webui/react/src/pages/ExperimentDetails/TrialsComparisonModal.tsx

```diff
@@ -219,147 +222,160 @@ export const TrialsComparisonTable: React.FC<TableProps> = ({
         (trials === undefined || trials.length === 0)
       ) ? (
         <Spinner center spinning={!isLoaded}>
-          <div className={[css.row, css.sticky].join(' ')}>
-            <div className={[css.cell, css.blank, css.sticky].join(' ')} />
-            {trialsDetails.map((trial) => (
-              <div className={css.cell} key={trial.id}>
-                <Tag
-                  className={css.trialTag}
-                  closable={!!onUnselect}
-                  onClose={() => handleTrialUnselect(trial.id)}>
-                  <Link path={paths.trialDetails(trial.id, trial.experimentId)}>
-                    {Array.isArray(experiment) ? (
-                      <Typography.Paragraph ellipsis={{ tooltip: true }}>
-                        {experimentMap[trial.experimentId]?.name}
-                      </Typography.Paragraph>
-                    ) : (
-                      `Trial ${trial.id}`
-                    )}
-                  </Link>
-                </Tag>
-              </div>
-            ))}
-          </div>
-          <div className={css.row}>
-            <div className={[css.cell, css.sticky, css.indent].join(' ')}>State</div>
-            {trialsDetails.map((trial) => (
-              <div className={css.cell} key={trial.id}>
-                <Badge state={trial.state} type={BadgeType.State} />
-              </div>
-            ))}
-          </div>
-          {Array.isArray(experiment) && (
-            <>
-              <div className={css.row}>
-                <div className={[css.cell, css.sticky, css.indent].join(' ')}>Experiment ID</div>
+          <table>
+            <thead>
+              <tr>
+                <th />
                 {trialsDetails.map((trial) => (
-                  <div className={css.cell} key={trial.id}>
-                    {trial.experimentId}
-                  </div>
+                  <th key={trial.id}>
+                    <Tag
+                      className={css.trialTag}
+                      closable={!!onUnselect}
+                      style={{ width: '100%' }}
+                      onClose={() => handleTrialUnselect(trial.id)}>
+                      <Link path={paths.trialDetails(trial.id, trial.experimentId)}>
+                        {Array.isArray(experiment) ? (
+                          <Typography.Text ellipsis={{ tooltip: true }}>
+                            {experimentMap[trial.experimentId]?.name}
+                          </Typography.Text>
+                        ) : (
+                          `Trial ${trial.id}`
+                        )}
+                      </Link>
+                    </Tag>
+                  </th>
                 ))}
-              </div>
-              <div className={css.row}>
-                <div className={[css.cell, css.sticky, css.indent].join(' ')}>Trial ID</div>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>State</td>
```

Comment:
nit: for semantics, this `td` (along with the other row titles) should be replaced with a `<th scope="row">`